e2e_tests/cli/test_push.py

```diff
@@ -110,3 +110,170 @@ def test_push_videos_with_non_native_fps(
+    }
+    expected_file_names = [
+        "image_1.jpg",
+        "image_2.jpg",
+        "image_3.jpg",
+        "image_4.jpg",
+        "image_5.jpg",
+        "image_6.jpg",
+    ]
+    push_dir = (
+        Path(__file__).parents[1] / "data" / "push" / "flat_directory_of_6_images.zip"
+    )
+    merge_mode = "channels"
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        tmp_dir = Path(tmp_dir_str)
+        with zipfile.ZipFile(push_dir) as z:
+            z.extractall(tmp_dir)
+            result = run_cli_command(
+                f"darwin dataset push {local_dataset.name} {tmp_dir}/flat_directory_of_6_images --item-merge-mode {merge_mode}"
+            )
+            assert_cli(result, 0)
+            wait_until_items_processed(config_values, local_dataset.id)
+            items = list_items(
+                config_values.api_key,
+                local_dataset.id,
+                config_values.team_slug,
+                config_values.server,
+            )
+            assert len(items) == 1
+            multi_channel_item = items[0]
+            assert multi_channel_item["name"] == expected_name
+            assert multi_channel_item["slot_types"] == expected_slot_types
+            assert multi_channel_item["layout"] == expected_layout
+            for num, slot in enumerate(multi_channel_item["slots"]):
+                assert slot["slot_name"] == expected_file_names[num]
+                assert slot["file_name"] == expected_file_names[num]
+
+
+def test_push_dicom_series(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Test pushing a multi-file DICOM item with the CLI. Check the resulting item is
+    structured as expected
+    """
+    expected_name = "flat_directory_of_2_dicom_files"
+    expected_slot_types = [
+        "dicom",
+        "dicom",
+    ]
+    expected_layout = {
+        "slots_grid": [
+            [
+                [
+                    "flat_directory_of_2_dicom_files",
+                ]
+            ]
+        ],
+        "version": 3,
+    }
+    expected_file_names = [
+        "flat_directory_of_2_dicom_files",
+    ]
+    push_dir = (
+        Path(__file__).parents[1]
+        / "data"
+        / "push"
+        / "flat_directory_of_2_dicom_files.zip"
+    )
+    merge_mode = "series"
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        tmp_dir = Path(tmp_dir_str)
+        with zipfile.ZipFile(push_dir) as z:
+            z.extractall(tmp_dir)
+            result = run_cli_command(
+                f"darwin dataset push {local_dataset.name} {tmp_dir}/flat_directory_of_2_dicom_files --item-merge-mode {merge_mode}"
+            )
+            assert_cli(result, 0)
+            wait_until_items_processed(config_values, local_dataset.id)
+            items = list_items(
+                config_values.api_key,
+                local_dataset.id,
+                config_values.team_slug,
+                config_values.server,
+            )
```

Comment:
The logic here repeats in every test. Maybe you can de-duplicate it. At least it is easier to dot it from line 261